addons/rook/1.10.6/install.sh

```diff
@@ -259,30 +259,6 @@ function rook_cluster_deploy_upgrade() {
         return 0
     fi
 
-    # 4. https://rook.io/docs/rook/v1.6/ceph-upgrade.html#4-wait-for-the-upgrade-to-complete
```

Comment:
We are just moving it for the end of the method.
See the rook doc  https://rook.io/docs/rook/v1.6/ceph-upgrade.html#4-wait-for-the-upgrade-to-complete it should be done at the end.